frontends/web/src/routes/account/info/signingconfiguration.tsx

```diff
@@ -59,6 +59,19 @@ class SigningConfiguration extends Component<Props, State> {
         apiPost(`account/${this.props.code}/verify-extended-public-key`, index);
     }
 
+    private scriptTypeTitle = (scriptType: string) => {
+        switch (scriptType) {
+            case 'p2pkh':
+                return 'Legacy';
+            case 'p2wpkh-p2sh':
+                return 'Segwit';
```

Comment:
SegWit, capital W? Official abbreviation: https://en.bitcoin.it/wiki/Segregated_Witness

Reply:
Using Segwit everywhere else (guides, etc).